doc/btp_sdp.txt

```diff
@@ -0,0 +1,62 @@
+SDP Service (ID 31)
+=====================
+
+Commands and responses:
+
+        Opcode 0x00 - Error response
+
+        Opcode 0x01 - Read Supported Commands command/response
+
+                Controller Index:       <controller id>
```

Comment:
read supported commands should be `<non-controller>`   (yes, there is few services where this is not followed, those will be fixed eventually)